packages/target-nodejs-sdk/test/api.spec.js

```diff
@@ -0,0 +1,303 @@
+                meta: {
+                  "experience.id": "1",
+                  "geo.connectionSpeed": "broadband",
+                  "activity.name": "vegas mbox",
+                  "profile.tntId": "b68417fa4c2c43afa065946ad59bf445",
+                  "activity.id": "543937",
+                  "experience.name": "Experience B",
+                  "option.name": "Offer3",
+                  "geo.domainName": "veracitynetworks.com",
+                  "option.id": "3",
+                  "profile.current_property": "no_property",
+                  "profile.isFirstSession": "true",
+                  "geo.country": "united states",
+                  "offer.name":
+                    "/vegas_mbox/experiences/1/pages/0/zones/0/1617913012293",
+                  "profile.marketingCloudVisitorId":
+                    "67987995057387431273176504937695227658",
+                  "profile.activeActivities": "543937",
+                  "activity.decisioningMethod": "server-side",
+                  "offer.id": "753664"
+                },
+                data: {
+                  id: "753664",
+                  format: "application/json",
+                  content: {
+                    hotel: "Circus Circus"
+                  }
+                }
+              }
+            ]
+          }
+        ]
+      },
+      {
+        type: "consent:preferences",
+        payload: [
+          {
+            collect: {
+              val: "y"
+            }
+          }
+        ]
+      },
+      {
+        type: "state:store",
+        payload: [
+          {
+            key: "kndctr_6FC947105BB267B70A495EE9_AdobeOrg_identity",
+            value:
+              "CiY1Mjc3NzEwODUxMDk3ODY4ODI1MDUwMTYyMDEwMjA3MzQ3Nzk5MFIQCOfst5i6LxABGAEqA09SMvAB5-y3mLov",
+            maxAge: 34128000
+          },
+          {
+            key: "kndctr_6FC947105BB267B70A495EE9_AdobeOrg_consent",
+            value: "general=in",
+            maxAge: 15552000
+          },
+          {
+            key: "kndctr_6FC947105BB267B70A495EE9_AdobeOrg_consent_check",
+            value: "1",
+            maxAge: 7200
+          }
+        ]
+      }
+    ]
+  };
+
+  const sessionId = "dummy_session_id";
+  const clientCode = "targetdataplatform";
+  const organizationId = "6FC947105BB267B70A495EE9@AdobeOrg";
+  const edgeConfigId = "34610d20-3c46-4636-b22f-eb87110dfb25:dev";
+  const requestId = "244f07a4895a47ef94a003bfb578d773";
+
+  const request = {
+    requestId,
+    execute: {
+      id: {
+        marketingCloudVisitorId: "52777108510978688250501620102073477990"
+      },
+      mboxes: [
+        {
+          name: "vegas",
+          index: 0
+        }
+      ]
+    }
+  };
+
+  const expectedResponse = {
+    visitorState: {
+      "6FC947105BB267B70A495EE9@AdobeOrg": expect.any(Object)
+    },
+    request: {
+      requestId: "244f07a4895a47ef94a003bfb578d773",
+      context: {
+        channel: "web",
+        timeOffsetInMinutes: -360
```

Comment:
Without this fix the test fails on my machine:
```suggestion
        timeOffsetInMinutes: expect.any(Number)
```

Reply:
Good catch, forgot to change that